api-inference-community/api_inference_community/validation.py

```diff
@@ -114,9 +115,18 @@ class QuestionInputsCheck(BaseModel):
     context: str
 
 
+class SimilarityFunctionEnum(str, Enum):
+    default = "default"
```

Comment:
What is the default ? Isn't it `cosine_similarity`? 

Or maybe it should just be `None` if we can't force it.  `default` is always an issue, because it's unclear what it is.

I am not aware of any model than can be used with `both` or `either` scheme, so I feel it should be the framework's responsability to actually handle this (hence, NOT being a parameter, just like the tokenization is not a choice of the pipeline)